RPS src/RPScomp.js

```diff
@@ -0,0 +1,24 @@
+//"stone_PNG13612" | "Paperball_yellow" | "scissors2"
+
+
+export default function rpsCompair(human, computer) {
+
+    if(human === 2 && computer === 'Paperball_yellow') {
+        return 1;
+    } else if(human === 0 && computer === 'scissors2') {
+        return 1;
+    } else if(human === 1 && computer === 'stone_PNG13612') {
```

Comment:
this function need to be refactored for readability and separation of concerns. 
`if(human === 'rock' && computer === 'paper') {
     return true (or 'win')
}`

but make sure to follow the lab instructions, as you were asked to make a getThrow() function and a getThrowFromNumber() function.